### ISSUE TITLE
Is master-data actually public ?

### ISSUE THREAD

**Issue by madushankaf** [tross-tyson/mdata_go#6]
`publicly share master data to support the private transactions.`

Can we call master-data actually public?  Wouldn't there be a requirement to have master-data only visible to selected participants? 

Just calling this out. I personally believe it should be public. More you expose your product, you will get more business :) 

better to check with other partners as well

**Comment by txross** [comment]
It's not all master data that we are sharing, just the basic products by their identification number.  I would make the argument that sharing identification numbers does not expose confidential data, which is why I purposely did not put a "description" in the data model. This shared master data is purely for supporting private transactions while enabling full consortium trust through exposing a list that is public to all members.

Let us connect with our business partners and gather the answers to these questions:

- If there is a new product that is not yet available to all customers, do we want to publicly share that product's GTIN on the blockchain?
- If there is a new product, will that product already have a GTIN at the point in time a manufacturer wants to "canary deploy" the product to specific customers?

**Comment by txross** [comment]
Quote from my Sales and Growth VP:

> As far as visibility to the GTIN/product information, I would think that customers should only have access to GTIN’s/products that they buy.

**Comment by madushankaf** [comment]
Yeah, that is the general feeling I also had. We may need confidentiality in other terms some kinda PIKE like implementation on your GO programs. 

Or else cut-over to Sabre at this point so the private transaction impl is easy, not easier though ;)
